machines/cmake_macros/intel-oneapi_betzy.cmake

```diff
@@ -0,0 +1,32 @@
+set(PIO_FILESYSTEM_HINTS "lustre")
+string(APPEND CFLAGS " -qopt-report -march=core-avx2")
+string(APPEND FFLAGS " -qopt-report -march=core-avx2")
+set(NETCDF_C_PATH	$ENV{EBROOTNETCDF})
+set(NETCDF_FORTRAN_PATH $ENV{EBROOTNETCDFMINFORTRAN})
+set(PNETCDF_PATH	$ENV{PNETCDF})
+
+if(DEFINED ENV{PIO})
+  set(PIO_LIBDIR "$ENV{PIO}/lib")
+  set(PIO_INCDIR "$ENV{PIO}/include")
+endif()
+
+string(REPLACE "-mkl=cluster" "" SLIBS "${SLIBS}")
+string(APPEND CPPDEFS " -DHAVE_GETTID")
+
+if (COMP_WAV STREQUAL ww3dev)
```

Comment:
This must now be ww3 NOT ww3dev. This change recently came into CIME and ccs_config.

Reply:
Done